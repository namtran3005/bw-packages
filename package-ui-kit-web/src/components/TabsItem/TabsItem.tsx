import * as React from 'react';

interface TabsItemProps {
  children?: any; // eslint-disable-line @typescript-eslint/no-explicit-any
  label: React.ReactChild;
  disabled?: boolean;
}

const TabsItem: React.FunctionComponent<TabsItemProps> = ({
  /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
  label,
  /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
  disabled,
  ...cleanProps
}) => <div {...cleanProps} />;

export { TabsItem, TabsItemProps };
