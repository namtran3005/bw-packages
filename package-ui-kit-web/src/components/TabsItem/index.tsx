export { TabsItem, TabsItemProps } from './TabsItem';
