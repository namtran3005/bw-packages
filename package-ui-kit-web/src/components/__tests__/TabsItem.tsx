import * as React from 'react';
import { shallow } from 'enzyme';
import { TabsItem } from '../TabsItem';

const defaultProps = {
  label: 'First Tab',
};

describe('TabsItem', () => {
  it('should render TabsItem', () => {
    const wrapper = shallow(<TabsItem {...defaultProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
