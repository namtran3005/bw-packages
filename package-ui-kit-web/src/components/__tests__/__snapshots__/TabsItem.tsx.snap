// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabsItem should render TabsItem 1`] = `<div />`;
